go/p2p/config/config.go

```diff
@@ -0,0 +1,112 @@
+// Package config implements global configuration options.
+package config
+
+import "time"
+
+// Config is the P2P configuration structure.
+type Config struct {
+	// Port to use for incoming P2P connections.
+	HostPort uint16 `yaml:"port"`
```

Comment:
Why not just `Port`?

Reply:
`Port` makes more sense, `HostPort` is what the original author named it.